get-started/index.md

```diff
@@ -56,127 +56,156 @@ redirect_from:
+libraries, environment variables, and configuration files. You can find
+thousands of images on [Docker Hub](https://hub.docker.com/explore/).
 
 A **container** is a runtime instance of an image&#8212;what the image becomes
-in memory when actually executed. It runs completely isolated from the host
-environment by default, only accessing host files and ports if configured to do
+in memory when executed. By default, a container runs completely isolated from
+the host environment, only accessing host files and ports if configured to do
 so.
 
-Containers run apps natively on the host machine's kernel. They have better
-performance characteristics than virtual machines that only get virtual access
-to host resources through a hypervisor. Containers can get native access, each
-one running in a discrete process, taking no more memory than any other
-executable.
+The use of Linux containers to build, deploy, and run applications is called
+[containerization](https://en.wikipedia.org/wiki/Operating-system-level_virtualization).
+Containers are not new but their use for easily deploying applications is.
 
 ## Containers vs. virtual machines
 
-Consider this diagram comparing virtual machines to containers:
+Each **virtual machine** (VM) runs a "guest" operating system (OS) with only
+virtual access to host resources through a hypervisor. VMs are resource
+intensive, and the resulting disk image and application state is an entanglement
+of OS settings, system-installed dependencies, OS security patches, and other
+easy-to-lose, hard-to-replicate ephemera. VMs provide an environment with far
+more resources than most applications need.
 
-### Virtual Machine diagram
+By contrast, a **container** runs applications natively and shares the kernel of
+the host machine with other containers, keeping it lightweight. Each container
+runs in a discrete process, taking no more memory than any other executable. The
+only information needed in a container image is the executable and its package
+dependencies, which never need to be installed on the host system. Because
+containers house their dependencies, a containerized app is portable and “runs
+anywhere.”
 
-![Virtual machine stack example](https://www.docker.com/sites/default/files/VM%402x.png)
+![Container stack example](https://www.docker.com/sites/default/files/Container%402x.png){:width="300px"} | ![Virtual machine stack example](https://www.docker.com/sites/default/files/VM%402x.png){:width="300px"}
 
-Virtual machines run guest operating systems&#8212;note the OS layer in each
-box. This is resource intensive, and the resulting disk image and application
-state is an entanglement of OS settings, system-installed dependencies, OS
-security patches, and other easy-to-lose, hard-to-replicate ephemera.
+## Prepare your Docker environment
 
-### Container diagram
+Install a [supported version](https://docs.docker.com/engine/installation/#supported-platforms){:target="_blank"}
```

Comment:
If you don't add `class="_"` after the target, the rest of the file is going to be shown in italic in any editor, which is kind of annoying side effect.

Reply:
This is not working for me. Have you seen this work? 

`[supported platform](https://docs.docker.com/engine/installation/#supported-platforms){:target class="_"}`

Reply:
`{:target="_blank" class="_"}`